src/watch/hearRingiOSWatchOS/hearRingiOSWatchOS WatchKit Extension/NotificationController.swift

```diff
@@ -11,13 +11,16 @@ import UserNotifications
 
 class NotificationController: WKUserNotificationHostingController<NotificationView> {
 
+    let alarms = Alarm()
     override var body: NotificationView {
         return NotificationView()
     }
 
     override func willActivate() {
         // This method is called when watch view controller is about to be visible to user
         super.willActivate()
+        //alarms.getAlarms()
```

Comment:
i realized i forgot to take these comments out, but i can remove them in my next alarm PR. if whoever reviews this has a problem with this ill get rid of em though

Reply:
oh nvm i will take em out i dont need any of these changes anymore